cms/templatetags/cms_tags.py

```diff
@@ -81,7 +90,12 @@ def _get_page_by_untyped_arg(page_lookup, request, site_id):
                     return page
         else:
             pages = get_page_queryset(site, draft=_use_draft(request))
-            return pages.select_related('node').get(**page_lookup)
+            try:
+                return pages.select_related('node').get(**page_lookup)
+            except Page.MultipleObjectsReturned:
+                messages.warning(request, warning_message)
+                logger.warning(warning_message)
+                return pages.select_related('node').filter(**page_lookup).first()
```

Comment:
I am uncertain if returning the first page is a good solution. Something went wrong here, and I'd stay with `None`. Clearly, you cannot predict which of the multiple pages will be first.